content/user/ui/report/Org-stats.md

```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Organization Statistics"
+description: "Overview of organization statistics charts"
+tags: "TeamServer licenses applications servers organizations report"
+-->
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics. 
+
+Go go to the Reports page via the User Menu to find information about licensing and breakdowns across various data points for onboarded applications; deployed servers and how they’re being utilized; and licensing to understand utilization and expirations. A collection of filters and charts let you decide which data you want to see and how you want to see it. 
+
+<a href="assets/images/org-stats-reports-page.png" rel="lightbox" title="Organization statistics on Reports page"><img class="thumbnail" src="assets/images/org-stats-reports-page.png"/></a>
+
+In the Licenses chart, view the number of overall licenses for Assess and Protect, as well as the the number of unlicensed applications and servers that exist in your organization. Click the application count to go to the Trial/Unlicensed quick view in the Applications page. 
```

Comment:
"Clicking the application count will navigate you to the Unlicensed quick view on the Applications page."